src/transformStateWithClones.js

```diff
@@ -7,7 +7,28 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const historyArray = [];
+  let changedObject = { ...state };
```

Comment:
Good comment, BTW 🙂 
Let's fix it as well - naming is a very important part of code